pytest/tests/sanity/validator_switch_key_quick.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+# Starts three validating nodes and one non-validating node
+# Set a new validator key that has the same account id as one of
+# the validating nodes. Stake that account with the new key
+# and make sure that the network doesn't stall even after
+# the non-validating node becomes a validator.
+
+import sys, time
+import pathlib
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+
+from cluster import start_cluster
+
+EPOCH_LENGTH = 20
+TIMEOUT = 100
+NUM_VALIDATORS = 2
+
+client_config = {
+    "tracked_shards": [0],  # Track all shards
+    "state_sync_enabled": True,
+    "store.state_snapshot_enabled": True
+}
+# It is important for the non-validating node to track all shards as well.
+# It needs to have all the data when it switches to validator role.
```

Comment:
Just for my context how does loading tries to memory work in this scenario? Would the new node need to load all the tries at first and would it drop the ones that are not needed once it becomes a validator? We should avoid the node needing to have enough memory for all shards forever. It doesn't need to happen in this PR but I'm curious about your thoughts on that.

Reply:
New node only needs to load data for the shards it will be validating. I will try to modify pytest to reflect that.